### DISCUSSION TITLE
WSA 2308 Update

### DISCUSSION THREAD

**Discussion by jaholme** [microsoft/WSA#421] [Announcements]
# What's New

We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2308.40000.1.0) includes improvements to platform reliability and functionality improvements.

## Release Notes

- Platform reliability improvements
- Localhost loopback support for "Local network access" experimental feature
- Clipboard reliability and performance improvements
- Kernel update to improve compatibility with running ARM applications on x64 processors 
- Correctly report isInMultiWindowMode=false when an app is fullscreen

# Giving feedback

If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5).

Thanks,
Windows Subsystem for Android™ Team

**Comment by HotCakeX** [comment]
Thank you for the update,

By the way this item

* Kernel update to improve compatibility with running ARM applications on x64 processors

Doesn't include fix for Microsoft Edge browser on WSA, right? because it still isn't working on WSA 2308.40000.1.0

https://github.com/microsoft/WSA/discussions/374#discussioncomment-6612894
https://github.com/microsoft/WSA/issues/249#issuecomment-1460660191

**Comment by bubbleguuum** [comment]
UDP multicast used for mDNS and SSDP discovery of devices on the local network is still not working:

#378

**Comment by yk00365** [comment]
When the wsa is supporting the ipv6 connections? The ipv6 connections is still not working.
https://github.com/microsoft/WSA/issues/389
